### PR TITLE
Give clearer expectations and submission instructions

### PR DESCRIPTION
**Pull request by nfelger**
- Any specific expectations for the frontend challenge (in addition to the ones from the b/e challenge)?
- Any expectations that we should remove b/c they're not a fit here?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,19 +4,45 @@
...
-Git during building your application please make sure to include the Git history as well.
+Use a non-proprietary tech stack of your choice and explain in a readme how to run your solution. Please use version control (git is preferred).
+
+## Submitting your solution
+
+Please submit your solution at least 2-3 days before your technical interview. You have two options:
+
+1. Hosted repository (preferred)
+
+Email a link to a GitHub/Bitbucket/GitLab/etc. repo to [challenge@digitalservice.bund.de](mailto:challenge@digitalservice.bund.de). Make sure to make the repo private 3 days after the interview.
```
Comment (carhartl) [inline README.md]: How about "Please make sure..."

File: README.md [hunk 2]
```diff
@@ -4,19 +4,45 @@
...
+
+
+2. Email 
+
+You can also email your solution as a zip file to [challenge@digitalservice.bund.de](mailto:challenge@digitalservice.bund.de). Make sure to include the commit history in your submission.
```
Comment (carhartl) [inline README.md]: How about "Please make sure..."

File: README.md [hunk 3]
```diff
@@ -4,19 +4,45 @@
...
 
 ## Evaluation
 
-You should turn in a complete and runnable solution, but do not gold-plate it: we don't expect you to spend more than a few hours on this exercise. We want to see that you know how to use APIs efficiently, how you structure and how you are testing your application. It should be easy to tell from the dashboard which ministries have provided the most data.
+We want to use this challenge to understand how you would approach such a problem in a work setting. Here are some specifics of what we are looking for:
+
+Among other things, we pay attention to:
```
Comment (carhartl) [inline README.md]: Seeing this again, I'm starting to wonder what are those other things? That might be too vague and lead to needless angst? The list of things is already quite long. How about omitting it altogether: "We are going to pay attention to:"

That is, put everything that is relevant to us into the list, but don't come up with other unknown/unexpected evaluation criteria.

File: README.md [hunk 4]
```diff
@@ -4,19 +4,45 @@
...
+- what your commits look like,
+- whether you know how to consume APIs efficiently,
+- what tooling you use,
+- overall consistency & attention to detail.
+
+You should turn in a complete and runnable solution, but do not gold-plate it: we don't expect you to spend more than a few hours on this exercise.
 
 In the technical interview that follows we would like to discuss your approach, and that includes talking about areas you would want to improve if given more time.
 
+Note that the UI won't be part of the evaluation (could be plain HTML for instance).
```
Comment (carhartl) [inline README.md]: I think this was copied from the backend challenge? I think here we want to evaluate the UI? 😬

File: README.md (hunk 3 above)
Comment (joschka) [reply README.md]: +1 Omit it and extend list as needed.

File: README.md [hunk 5]
```diff
@@ -4,19 +4,45 @@
...
+
+Among other things, we pay attention to:
+- how you structure your application and your source code,
+- how you name variables/functions/classes/packages,
+- how you test your application and what kinds of tests you're writing,
+- how you document your application, including instructions to run the project,
+- what your commits look like,
+- whether you know how to consume APIs efficiently,
+- what tooling you use,
+- overall consistency & attention to detail.
```
Comment (joschka) [inline README.md]: I tend to take these out:
`how you name variables/functions/classes/packages`
(either I have to extend the list, e.g. "css classes"; or assume naming as part of structure)
`whether you know how to consume APIs efficiently`
(this is mostly part of the backend challenge, right?)

and add these ones:
`a robust understanding of HTML, CSS and JavaScript`
(my intention is to highlight, that we have these three areas: document/semantic markup + presentation/formatting + functionality)
`an accessible and usable solution`
(understanding of accessibility is very important + basics in UI + UX for good collaboration with UI and UX designers)

File: README.md (hunk 4 above)
Comment (joschka) [reply README.md]: +1 :)

File: README.md [hunk 6]
```diff
@@ -4,19 +4,45 @@
...
+
+
+2. Email 
+
+Please make sure to also email your solution as a zip file to [challenge@digitalservice.bund.de](mailto:challenge@digitalservice.bund.de). Make sure to include the commit history in your submission.
 
 ## Evaluation
 
-You should turn in a complete and runnable solution, but do not gold-plate it: we don't expect you to spend more than a few hours on this exercise. We want to see that you know how to use APIs efficiently, how you structure and how you are testing your application. It should be easy to tell from the dashboard which ministries have provided the most data.
+We want to use this challenge to understand how you would approach such a problem in a work setting. Here are some specifics of what we are looking for:
```
Comment (joschka) [inline README.md]: The `Here are some specifics of what we are looking for:` is redundant when we have in the next line `We are going to pay attention to:`

File: README.md (hunk 5 above)
Comment (nfelger) [reply README.md]: > `how you name variables/functions/classes/packages`
> (either I have to extend the list, e.g. "css classes"; or assume naming as part of structure)

Works for me.

> whether you know how to consume APIs efficiently
> (this is mostly part of the backend challenge, right?)

👍 

> and add these ones: [...]

Awesome!

File: README.md [hunk 7]
```diff
@@ -4,16 +4,40 @@
...
+
+Please submit your solution at least 2-3 days before your technical interview. You have two options:
+
+1. Hosted repository (preferred)
+
+Please make sure to email a link to a GitHub/Bitbucket/GitLab/etc. repo to [challenge@digitalservice.bund.de](mailto:challenge@digitalservice.bund.de). Make sure to make the repo private 3 days after the interview.
+
+2. Email
+
+Please make sure to also email your solution as a zip file to [challenge@digitalservice.bund.de](mailto:challenge@digitalservice.bund.de). Make sure to include the commit history in your submission.
```
Comment (nfelger) [inline README.md]: @chrkaatz I think we now ended up with a contradiction (my earlier phrasing also could be read in that way): we say "you have two options" followed by "please do 1." and "please do 2.".

Do you think people should do both? I think we just want them to do one. If so, how about this:

```suggestion
Please submit your solution at least 2-3 days before your technical interview. You have two options:

1. Hosted repository (preferred)

Please email a link to a GitHub/Bitbucket/GitLab/etc. repo to [challenge@digitalservice.bund.de](mailto:challenge@digitalservice.bund.de). Make sure to make the repo private 3 days after the interview.

2. Email

Alternatively, you can email your solution as a zip file to [challenge@digitalservice.bund.de](mailto:challenge@digitalservice.bund.de). Make sure to include the commit history in your submission.
```

Comment (nfelger) [reply README.md]: @chrkaatz You accepted it on the backend-challenge, so I'm assume that counts here as well :)

**Comment by nfelger** [comment]
@joschka I think this is ready to merge now, do you want to take a last look and merge if it's ok from your end?

**Pull request merged**